### ISSUE TITLE
The Unofficial builds of VSCode are not supported..

### ISSUE THREAD

**Issue by SscSPs** [codercom/sshcode#29]
https://github.com/codercom/sshcode/blob/cb036d8b8c45a43ebc804e085202f4feff84e8d1/main.go#L307
Path to the config directory is hardcoded here ( `$HOME/.config/Code/User/` ),
the unofficial builds doesn't use this path,
they use  `$HOME/.config/Code - OSS/User/` instead. 
(I'm not sure about Mac paths)

We should probably support them as well.

EDIT: 
Also, VSCodium uses `$HOME/.config/VSCodium/User/`

**Comment by jeasonstudio** [comment]
@SscSPs 
`$HOME/.config/Code - OSS/User/` is the dev config folder for vscode on Linux. So this will not cause any problems.  
However, how do `Insiders` version users will deal with these? @emerose

**Comment by SscSPs** [comment]
@jeasonstudio it causes issue while syncing, 
```
2019-04-22 18:04:48 INFO        syncing settings
sending incremental file list
rsync: change_dir "/home/sahil/.config/Code/User" failed: No such file or directory (2)

sent 20 bytes  received 12 bytes  12.80 bytes/sec
total size is 0  speedup is 0.00
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1189) [sender=3.1.3]
2019-04-22 18:04:50 FATAL       failed to sync settings: failed to rsync '/home/sahil/.config/Code/User/' to 'myserver:.local/share/code-server/User/': exit status 23
```

thats because the path doesn't exist

**Comment by ammario** [comment]
Fixed in dcd29f0
